tasks/winbuildscripts/Update-Winget.ps1

```diff
@@ -1,17 +1,25 @@
 $ErrorActionPreference = 'Stop';
 Set-Location c:\mnt
 
-Invoke-WebRequest https://aka.ms/wingetcreate/latest -OutFile wingetcreate.exe
+(New-Object System.Net.WebClient).DownloadFile("https://aka.ms/wingetcreate/latest", ".\wingetcreate.exe")
```

Comment:
Why don't we download it from https://github.com/microsoft/winget-create/releases/download/v0.4.0.3-preview/wingetcreate.exe instead? That way we can pin the version